README.md

```diff
@@ -51,5 +61,5 @@ There are two ways to deploy your app to Netlify, you can either link your app t
 netlify deploy --build
 
 # production deployment
-netlify deploy --build --prod
+netlify deploy --build --prodnpx
```

Comment:
`--prodnpx` is missing from the [Netlify CLI](https://cli.netlify.com/commands/deploy/#deploy) docs - is there context that you can give on what this flag does?